src/main/java/seedu/address/security/User.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.security;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/***
+ * Represents the authenticated User
+ */
+public class User {
+    private Model model;
+    private ObservableList<Person> list;
+    private String username;
+    private Person person;
+
+    public User(String username, Model model) {
+        this.username = username;
+        this.model = model;
+        matchPerson(username);
+    }
+
+    /***
+     *
+     * Finds @param name inside database to match it to the current user
+     *
+     */
+    private void matchPerson(String name) {
+        list = model.getAddressBook().getPersonList();
+        //Loops through personlist to get matched name Person Class
+        list.forEach((Person) -> {
```

Comment:
```suggestion
        for (Person person : list) {
            if (...) {
                ...
            }
        }
```

Reply:
This way you get to avoid the weird Java lambda function thing

Reply:
Lambda functions are quite commonly used. But ok i guess if that works ill use that way